data/Lawes_2014/metadata.yml

```diff
@@ -0,0 +1,105 @@
+source:
+  primary:
+    key: Lawes_2014
+    bibtype: Article
+    year: 2014
+    author: Michael J. Lawes, Sarah J. Richardson, Peter J. Clarke, Jeremy J. Midgley,
+      Matt S. McGlone and Peter J. Bellingham
+    title: Bark thickness does not explain the different susceptibility of Australian
+      and New Zealand temperate rain forests to anthropogenic fire
+    journal: Journal of Biogeography
+    volume: 41
+    number: 8
+    pages: 1467--1477
+    doi: 10.1111/jbi.12292
+contributors:
+  data_collectors:
+  - last_name: Lawes
+    given_name: Michael
+    ORCID: 0000-0002-2381-6147
+    affiliation: unknown
+    notes: acknowledge only
+  austraits_curators: Sophie Yang
+dataset:
+  data_is_long_format: no
+  custom_R_code: .na
+  collection_date: unknown/2014
+  taxon_name: species
+  location_name: location
+  description: A study comparing relative bark thickness, as an index of fire resistance,
+    of dominant canopy species between Australia and New Zealand cool temperate rain
+    forests, to evaluate why New Zealand forests were more susceptible to repeated
+    anthropogenic fires than Australian forests.
+  basis_of_record: field
+  life_stage: adult
```

Comment:
I think it is fine to leave this as adult - I suspect the vast majority of trees sampled were larger and it is possible that a small tree was already flowering.